universal-login-sdk/lib/api/sdk.ts

```diff
@@ -93,7 +93,7 @@ class UniversalLoginSDK {
   }
 
   async createFutureWallet() {
-    await this.getRelayerConfig();
+    await this.fetchRelayerConfig();
```

Comment:
use `getRelayerConfig()` to ensure it is loaded, but don't load it

Reply:
👍